### PR TITLE
Test and fix cops rendering through model8

### PR DESCRIPTION
**Pull request by jcorbin**
As it turns out: refactoring / perf tuning without sufficient test coverage is hazardous.

### CODE REVIEW

File: internal/cops/display/render_test.go [hunk 1]
```diff
@@ -44,3 +45,28 @@ func TestRenderBlankAndMultiRuneCellOver(t *testing.T) {
...
 	assert.Equal(t, []byte(" "+whiteHand+"\r\033[2C "), buf)
 }
+
+func TestRender_model8(t *testing.T) {
+	dis := New(image.Rect(0, 0, 16, 8))
+	for y := 0; y < 8; y++ {
+		b := Colors[y]
+		for x := 0; x < 16; x++ {
+			f := Colors[x/2]
+			dis.SetRGBA(x, y, "<", f, b)
```
Comment (jcorbin) [inline internal/cops/display/render_test.go]: going to generalize this test a bit moe; stay tuned

File: internal/cops/display/render_test.go [hunk 2]
```diff
@@ -44,3 +45,28 @@ func TestRenderBlankAndMultiRuneCellOver(t *testing.T) {
...
+			x++
+			dis.SetRGBA(x, y, ">", f, b)
+		}
+	}
+	// TODO also cur = Start
+	buf, _ := Render(nil, Reset, dis, Model3)
+	assert.Equal(t, []string{
+		"[30m<>[31m<>[32m<>[33m<>[34m<>[35m<>[36m<>[37m<>",
```
Comment (jcorbin) [inline internal/cops/display/render_test.go]: I love having literal escape codes btw, since it makes `git diff` actually show colored =D

**Comment by jcorbin** [comment]
Towards fixing #35

File: internal/cops/display/display.go [hunk 3]
```diff
@@ -94,14 +94,18 @@ func (d *Display) SetRGBA(x, y int, t string, f, b color.RGBA) {
...
 	d.Text.Strings[i] = t
-	d.Foreground.Pix[i] = f.R
-	d.Foreground.Pix[i+1] = f.G
-	d.Foreground.Pix[i+2] = f.B
-	d.Foreground.Pix[i+3] = f.A
-	d.Background.Pix[i] = b.R
-	d.Background.Pix[i+1] = b.G
-	d.Background.Pix[i+2] = b.B
-	d.Background.Pix[i+3] = b.A
+	j := i * 4
```
Comment (kriskowal) [inline internal/cops/display/display.go]: Ah, hah.

**Pull request merged**